File: matiasmasso/Mat.NET/DTO/DTOPost.cs
using System;

namespace DTO
{

    public class DTOPost : DTOBaseGuid
    {
        public string Title { get; set; }
        public string Excerpt { get; set; }
        public string Url { get; set; }
        public string ThumbnailUrl { get; set; }


        public DTOPost() : base()
        {
        }

        public DTOPost(Guid oGuid) : base(oGuid)
        {
        }
    }

}

File: matiasmasso/Mat.NET/DT2/DTOPost.vb
Public Class DTOPost
    Inherits DTOBaseGuid
    Property Title As String
    Property Excerpt As String
    Property Url As String
    Property ThumbnailUrl As String


    Public Sub New()
        MyBase.New
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub


End Class
